src/scene/load/x3dloadinternaltiledmap.pas

```diff
@@ -48,6 +48,39 @@ function LoadTiledMap2d(const Stream: TStream; const BaseUrl: String): TX3DRootN
   { Convert Tiled map into X3D node. }
   TCastleTiledMapConverter = class
   strict private
+    type
```

Comment:
Indent types within the `type` section by 2 additional spaces, also like in https://castle-engine.io/coding_conventions#_indenting_inside_classes .